mu4/userscores/internal/userscoresconfiguration.cpp

```diff
@@ -76,3 +77,13 @@ io::paths UserScoresConfiguration::templatesDirPaths() const
 
     return dirs;
 }
+
+QColor UserScoresConfiguration::templatePreviewBackgroundColor() const
```

Comment:
Yes, better to keep all module configuration in one place. For example, the notation background and the template background not necessarily have the same color, it may be different